runner_http.go

```diff
@@ -155,24 +155,20 @@ type reqMeta struct {
 	Context     json.RawMessage `json:"context"`
 	AccessToken string          `json:"access_token"`
 	Method      string          `json:"method"`
-	Params      struct {
-		Header http.Header `json:"header"`
-		Query  url.Values  `json:"query"`
-	} `json:"params"`
-	URL     string `json:"url"`
-	TraceID string `json:"trace_id"`
+	Header      http.Header     `json:"header"`
+	Query       url.Values      `json:"query"`
```

Comment:
Why are these singular here and multiple elsewhere?

Reply:
matched the previous naming. I can fix that up though, good question 👍